### DISCUSSION TITLE
How to handle different instance types

### DISCUSSION THREAD

**Discussion by AnthonyC-S** [talmolab/sleap#1645] [Help!] [answered]
Is there a recommendation on how to handle different types of instances? My experiment tracks a single rat from above but there are a few wooden blocks that are mostly stationary but change positions each experiment. It would be really helpful to be able to get the coordinates of each of the wooden blocks. Can I have different sets of nodes, i.e. rat body parts and block corners? Any recommendations on best way to proceed? Thank you in advance!

**Comment by Marielamend** [comment, marked as answer]
Hello, @AnthonyC-S!

To tackle this, you need to train two separate models with different skeletons (one consisting of rat body parts and the other consisting of block corners) and then run them in the same video. Make sure that each model with its respective skeleton is made in a separate project. This way you can get the coordinates you want for each instance. 


Any other questions let us know!

Have a great rest of your day,
Mariela
